src/docs/cookbook/testing/integration/introduction.md

```diff
@@ -325,31 +214,39 @@ and download the corresponding web driver:
   * Safari: Safari can only be tested on a Mac;
     the SafariDriver is already installed on Mac machines.
   * Edge [Download EdgeDriver][]
+-->
 
-Launch the WebDriver, for example: 
+To get started testing in a web browser, [Download ChromeDriver][].
 
-```shell
-./chromedriver --port=4444
+Next, create a new directory named `test_driver` containing a new file
+named`integration_test.dart`:
+
+```dart
+import 'package:integration_test/integration_test_driver.dart';
+
+Future<void> main() => integrationDriver();
 ```
-From the root of the project,
-run the following command:
+
+Launch WebDriver, for example: 
 
 ```shell
-flutter drive --target=test_driver/app.dart --browser-name=[browser name] --release
+chromedriver --port=4444
 ```
 
-To simulate different screen dimensions, you can use the `--browser-dimension` argument,
-for example:
+From the root of the project, run the following command:
 
 ```shell
-flutter drive --target=test_driver/app.dart --browser-name=chrome --browser-dimension 300,550 --release
+flutter drive \
+  --driver=test_driver/integration_test.dart \
+  --target=integration_test/app_test.dart \
+  -d web-server
 ```
 
-Will run the tests in the `chrome` browser in a window with dimensions 300 by 550.
-
 [Download ChromeDriver]: https://chromedriver.chromium.org/downloads
 [Download EdgeDriver]: https://developer.microsoft.com/en-us/microsoft-edge/tools/webdriver/
 [Download GeckoDriver]: https://github.com/mozilla/geckodriver/releases
 [flutter_driver]: {{site.api}}/flutter/flutter_driver/flutter_driver-library.html
+[integration_test]: {{site.github}}/flutter/flutter/tree/master/packages/integration_test
+[Integration testing]: /docs/testing/integration-tests
```

Comment:
Are you planning to update this page as well? There is a bit of duplicated info there that is also covered on this page

Reply:
Yes, I'm working on some changes to this page (updating to use `flutter test` for example). The cookbook articles often contain some duplicate info.

Reply:
This page is updated now